src/main/java/seedu/storemando/logic/parser/ReminderCommandParser.java

```diff
@@ -22,14 +24,22 @@
      * @throws ParseException if the user input does not conform the expected keyword
      */
     private long timeConversion(Long parsedNum, String timeUnit) throws ParseException {
-        String timeUnitLowerCase = timeUnit.toLowerCase();
-
-        if (timeUnitLowerCase.startsWith(DAY_KEYWORD)) {
-            return parsedNum;
-        } else if (timeUnitLowerCase.startsWith(WEEK_KEYWORD)) {
-            return parsedNum * 7;
+        if (parsedNum == 1 || parsedNum == -1 || parsedNum == 0) {
```

Comment:
Just a suggestion. Would it be better if you abstract the inner if-else clause into another function? Because both the if-else clause looks similar. Maybe, you can have the function takes in another argument to differentiate these 2 different if-else clause?

Reply:
Alright I will try :)